src/nvim/os/env.c

```diff
@@ -1,11 +1,22 @@
 // env.c -- environment variable access
 
+#include <assert.h>
```

Comment:
Given that this is a system include, it should not be adjacent to `#include <uv.h>`